benches/benchmarks.rs

```diff
@@ -1,8 +1,17 @@
 use criterion::black_box;
 use criterion::{criterion_group, criterion_main, Bencher, BenchmarkId, Criterion, Throughput};
-use rs_merkle_tree::store::{MemoryStore, RocksDbStore, SledStore, SqliteStore, Store};
+use rs_merkle_tree::store::Store;
 use rs_merkle_tree::{hasher::Keccak256Hasher, node::Node, tree::MerkleTree};
 
+#[cfg(feature = "memory_store")]
```

Comment:
can we keep the test/benchmark files clean (no #![cfg(...)])? maybe [this](https://doc.rust-lang.org/cargo/reference/cargo-targets.html#the-required-features-field) or the `--all-features` flag can help?